### DISCUSSION TITLE
ScalesYAt only can be set once?

### DISCUSSION THREAD

**Discussion by f3l1x-c** [Live-Charts/LiveCharts2#1378] [Q&A]
Hey i am currently trying to implement a realtime data visualization and i have two y axis. I want that the user can change the scale of each axis so he can decide if he want to display the graph on the left and right side. But if i want to set the ScalesYAt property of the LineSeries a second time it doesnt work is there a way i can change this without reflection?

**Comment by kendragoni** [comment]
I would also like to know if this can be changed once set.  I have not found a way yet.

**Comment by NicholasMerritt-eaton** [comment]
It may be necessary to have your <chart name>.Series be an ObservableCollection<ISeries> (or similar), so the chart knows when the series properties update.  Otherwise there's nothing specifically stopping the ScalesYAt property from changing an arbitrary amount of times.
